conf.d/unbound.yaml.example

```diff
@@ -0,0 +1,5 @@
+init_config:
+  # Not required for this check
```

Comment:
it might be worthwhile to add `sudo` as a flag along with a detailed comment for why it defaults to true?